gui/src/elm/Settings.elm

```diff
@@ -58,48 +82,125 @@ update msg model =
         NewRelease ( notes, name ) ->
             ( { model | newRelease = Just ( notes, name ) }, Cmd.none )
 
+        FillAddressAndDevice ( address, deviceName ) ->
+            ( { model | address = address, deviceName = deviceName }, Cmd.none )
+
+        UpdateDiskSpace disk ->
+            ( { model | disk = disk }, Cmd.none )
+
+        UnlinkCozy ->
+            ( { model | busyUnlinking = True }, unlinkCozy () )
+
+        CancelUnlink ->
+            ( { model | busyUnlinking = False }, Cmd.none )
+
 
 
 -- VIEW
 
 
-view : Helpers -> Model -> Html Msg
-view helpers model =
+humanReadableDiskValue : Helpers -> Float -> String
+humanReadableDiskValue helpers v =
+    (toString (round (v / 1000000)) ++ " M" ++ (helpers.t "Account b"))
+
+
+diskQuotaLine : Helpers -> Model -> Html Msg
+diskQuotaLine helpers model =
+    if model.disk.quota == 0 then
```

Comment:
Maybe introduce some `hasDiskQuota` function?
And reuse it so you don't need to repeat the div/text/values separator?